platform/security/idp/security-idp-client/src/main/resources/templates/soap.handlebars

```diff
@@ -12,10 +11,7 @@
  *
  **/
 -->
-<soapenv:Envelope
-        xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/"
-        xmlns:xsd="http://www.w3.org/2001/XMLSchema"
-        xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
+<soapenv:Envelope xmlns:soapenv="http://schemas.xmlsoap.org/soap/envelope/">
```

Comment:
Need to revert changes to this file since it is SOAP and those other namespaces are in use in the templated pieces.

Reply:
Thanks. Turns out the IntelliJ formatter removed them and I didn't notice.